runtime/src/main/java/org/corfudb/protocols/wireprotocol/MultipleReadRequest.java

```diff
@@ -0,0 +1,38 @@
+package org.corfudb.protocols.wireprotocol;
+
+import io.netty.buffer.ByteBuf;
+
+import java.util.Arrays;
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+
+/**
+ * Created by maithem on 7/28/17.
+ */
+@Data
+@AllArgsConstructor
+public class MultipleReadRequest implements ICorfuPayload<MultipleReadRequest> {
+    @Getter
+    final List<Long> addresses;
```

Comment:
What is the reason this is a list ? Is it because there is an implicit order to these addresses that we want to exploit now or later ?